it/uk/gov/hmrc/agentsubscriptionfrontend/stubs/AuthStub.scala

```diff
@@ -71,6 +71,42 @@ object AuthStub {
     sessionKeysForMockAuth(user)
   }
 
+  def isSubscribedToMtd(user: SampleUser): Unit = {
+    stubFor(get(urlEqualTo(user.enrolmentsLink))
+        .willReturn(
+          aResponse()
+              .withStatus(200)
+              .withBody(
+                s"""
+                   |[{"key":"HMRC-AS-AGENT","identifiers":[{"key":"Arn","value":"ARN0001"}],"state":"Activated"}]
```

Comment:
That's a very good question--I think it was originally a guess as we didn't have the q-pack. It works integrated with auth, which is probably why I didn't question it further---the changes to gg-stubs didn't include an identifier key for the enrolment at all.